taxcalc/functions.py

```diff
@@ -385,8 +391,9 @@ def AMED(e00200, MARS, AMED_thd, _sey, AMED_trt, FICA_mc_trt, FICA_ss_trt):
                            max(0, _sey) * (1 - 0.5 *
                                            (FICA_mc_trt + FICA_ss_trt)) -
                            AMED_thd[MARS - 1])
+    _fica = _fica + _amed
```

Comment:
Ref: http://www.irs.gov/taxtopics/tc751.html
Additional Medicare is part of Affordable Care Act (PPACA), which belongs to FICA